src/elf.rs

```diff
@@ -448,8 +448,8 @@ pub const EM_CYPRESS_M8C: u16 = 161;
 pub const EM_R32C: u16 = 162;
 /// NXP Semi. TriMedia
 pub const EM_TRIMEDIA: u16 = 163;
-/// QUALCOMM DSP6
-pub const EM_QDSP6: u16 = 164;
```

Comment:
Do you know the official source for these definitions? binutils still uses `EM_QDSP6`, and this is also what is listed at http://sco.com/developers/gabi/latest/ch4.eheader.html

Reply:
EM_HEXAGON is the name used by Qualcomm in their Linux and LLVM ports.